tests/v2/actions/services/verify.go

```diff
@@ -84,3 +94,94 @@ func VerifyAWSLoadBalancer(t *testing.T, client *rancher.Client, serviceLB *v1.S
 	})
 	require.NoError(t, err)
 }
+
+// VerifyClusterIP is a helper function that verifies the cluster is able to connect to the cluster ip service by ssh shell
+func VerifyClusterIP(client *rancher.Client, clusterName string, clusterID string, steveClient *steveV1.Client, serviceID string, path string, content string) error {
```

Comment:
[nit] typically you only send 1 of some of these vars:
* rancher.Client vs. steve.Client -> as you can get the steveclient from the rancherclient
* clusterName vs clusterID as you can get the other easily if you have one

we should keep function arguments to a minimum when possible.